com.unity.render-pipelines.high-definition/Runtime/Lighting/LightLoop/cleardispatchindirect.compute

```diff
@@ -1,5 +1,8 @@
 #pragma kernel ClearDispatchIndirect
 #pragma kernel ClearDrawProceduralIndirect
+
+#pragma multi_compile _ PLATFORM_LANE_COUNT_32
```

Comment:
Given that this is static per platform, why a multi compile vs defining PLATFORM_LANE_COUNT in relevant include files?

Reply:
For optimizing against specific GPU models, we need to be able to pick up PLATFORM_LANE_COUNT_32 during runtime based on the actual GPU model used (AMD vs Apple GPU)